witness/README.md

```diff
@@ -0,0 +1,44 @@
+Witnessing
+============
+
+Witnessing is an important part of any transparent ecosystem, as it ensures that
+misbehavior on the part of logs can be prevented or detected. This is done by
+ensuring that the _checkpoints_ published by logs are _consistent_, which means 
+the log is presenting a globally consistent view of its contents to everyone and
```

Comment:
"... is _consistent_, which means that it is globally consistent..." :)

Reply:
Who doesn't love a good tautology?  Changed this now :).